indico/modules/rb/tasks.py

```diff
@@ -111,9 +62,17 @@ def roombooking_occurrences():
 
     try:
         for occ in occurrences:
-            notify_upcoming_occurrence(occ)
-            occ.notification_sent = True
-            if occ.reservation.repeat_frequency == RepeatFrequency.DAY:
-                occ.reservation.occurrences.update({'notification_sent': True})
+            same_user_occurrences = []
+            for other in occurrences:
```

Comment:
FWIW, I think the number of queries in this task was awful before. Dunno if it's worth to optimize this a lot - there's so much lazy=dynamic in roombooking code that you can't really joinedload/subqueryload much anyway.